### ISSUE TITLE
"Skip to main content" link is not provided at top of page:A11y_ASP.NET_Movies_Create New_ByPassBlocks

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/aspnetcore#56313]
## GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #BM_ASP.NET_Web_Jun2024; #.NETCore; #WCAG2.4.1; #DesktopWeb; #Win11; #FTP; #ChromiumEdge; #ByPass Blocks;
## Environment Details:
App name: ASP.NET
Window Version: Win 11
OS Build: 22631.3737
Chromium Edge Version 126.0.2592.61 (Official build) (64-bit)
## Repro Steps:
1. Download the "RazorPagesApp" zip file and run .exe file.
2. Command prompt will open, TAB to "http://localhost:5000" link and press CTRL + ENTER key.
3. TAB to "Movies" tab and hit ENTER key. 
4. Now press TAB key to navigate main content keyboard focus will appear on first control of page. and user need to navigate with all links present in header.
5. Observe that "Skip to main content" link is not provided at top of page.
## Actual Result:
"Skip to main content" link is not provided at top of page. here when we open any link, we need to navigate with all links present in header to navigate main content.

**Simila issue observed in Razor Pages Unchanged app**
## Expected Result:
"Skip to main content" link should be provided at top of page. so, when we open any link, we no need to navigate with all links present in header to navigate main content.
## User Impact:
Keyboard only user will face difficulty when they open any link, they need to navigate with all links present in header again to navigate main content. 
## Attachment

https://github.com/dotnet/aspnetcore/assets/124254981/9eb0d4b0-07ec-46ca-ad8b-b316d526799e

Simila issue observed in Razor Pages Unchanged app

https://github.com/dotnet/aspnetcore/assets/124254981/5457a865-a3d4-4246-9c16-56fdf6374b5d

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by javiercn** [comment]
We looked into the issue, and we don't think we want to add this logic to the templates, as it's an enhancement and complicates the maintenance going forward.
